src/app/core/modal/modal.component.css

```diff
@@ -23,7 +23,7 @@
   max-height: 100%;
   width: 500px;
   margin-left: -250px;
-  height: 200px;
+  min-height: 200px;
```

Comment:
#261 to let the modal grow to the size of its content (but not larger than 100%)